tailwind.config.js

```diff
@@ -11,37 +10,34 @@ module.exports = {
     },
     extend: {
       colors: {
+        // Generated with https://www.tailwindshades.com/
         yellow: {
           500: "#F0AD4E",
         },
         red: {
           500: "#FF4A4A",
         },
         cyan: {
-          ...colors.cyan,
           500: "#5294AD",
           700: "#00739E",
         },
-        blueGray: {
-          ...colors.blueGray,
-          300: "#C2C9E0",
+
+        primary: {
+          DEFAULT: "#F0AD4E",
+        },
+        secondary: {
+          DEFAULT: "#151B1E",
         },
-        trueGray: colors.trueGray,
-        warmGray: colors.warmGray,
+        focus: {
+          DEFAULT: "#00739E",
+        },
+
+        info: "#292D3B",
+        success: "#479048",
+        warning: "#F0AD4E",
+        danger: "#EC6A5E",
       },
     },
   },
-  variants: {
-    extend: {
-      backgroundColor: ["checked"],
-      borderColor: ["checked"],
-      cursor: ["disabled"],
-      margin: ["first", "last"],
-      padding: ["first", "last"],
-      ringWidth: ["hover", "active"],
-      ringColor: ["hover", "active"],
-    },
-    opacity: ({ after }) => after(["disabled"]),
-  },

```

Comment:
wouldn't this be a violation of behavior based on these settings?

Reply:
All variants are applied by default now:
https://tailwindcss.com/docs/upgrade-guide#remove-variant-configuration